### PR TITLE
[WIP] Quotes

### PR DESCRIPTION
**Pull request by THECASE99**
The quotes are working, please merge it.

### CODE REVIEW

**Comment by Riscky** [comment]
What happens if the quote document does not exist? What if it is empty?

File: src/components/Tile.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+export default class Tile extends React.Component {
+    constructor(props) {
+        super(props);
+        this.props = props;
+    }
+
+    render() {
+        console.log(this.props);
```
Comment (Mstiekema) [inline src/components/Tile.js]: Remove log

File: package-lock.json [hunk 2]
```diff
@@ -2258,7 +2258,8 @@
         "ansi-regex": {
```
Comment (Riscky) [inline package-lock.json]: Please do not change this file like this

File: src/containers/App.js [hunk 3]
```diff
@@ -1,195 +1,268 @@
-import React, { Component, PropTypes } from 'react';
+import React, {Component, PropTypes} from 'react';
 import fetch from 'isomorphic-fetch';
 import Poster from '../components/Poster';
 import Activities from '../components/Activities';
 import Clock from '../components/Clock';
+// import Quotes from '../components/Quotes';
```
Comment (Riscky) [inline src/containers/App.js]: Remove unused line

File: style.css [hunk 4]
```diff
@@ -82,3 +83,42 @@ body {
 .activity.active {
 	background: #161616;
 }
+.quote {
+	background-image: url("src/img/tegelTemplate.jpg");
+	width:   100vw;
+	height:   100vh;
+	background-size: 100% 100%;
+
```
Comment (Mstiekema) [inline style.css]: Please remove empty white lines in css block

File: style.css [hunk 5]
```diff
@@ -82,3 +83,42 @@ body {
 .activity.active {
 	background: #161616;
 }
+.quote {
+	background-image: url("src/img/tegelTemplate.jpg");
```
Comment (Mstiekema) [inline style.css]: Missing file

File: src/containers/App.js [hunk 6]
```diff
@@ -1,195 +1,268 @@
...
 function setDate(activity) {
-  return Object.assign(
-    {},
-    activity,
-    { has_start_time : activity.start_date.indexOf('T') > -1 },
-    { has_end_time : activity.end_date && activity.end_date.indexOf('T') > -1 },
-    { start_date: new Date(activity.start_date) },
-    activity.end_date ? { end_date : new Date(activity.end_date) } : null
-  );
+    return Object.assign(
```
Comment (Riscky) [inline src/containers/App.js]: This reformat makes the diff very hard to read, please do this in a separate commit

**Comment by Mstiekema** [comment]
> What happens if the quote document does not exist? What if it is empty?

It displays the last activity or ad twice the time (16 seconds). A check should be in place to check if there's some JSON. 

Is JSON quotes available? -> Show quote
Not available? -> Skip current scene

File: package-lock.json (hunk 2 above)
Comment (Mstiekema) [reply package-lock.json]: Package lock gets updated by running npm install, these changes should be committed. I see nothing wrong with this.

**Comment by Mstiekema** [comment]
Decided to not merge this PR for now. We do want to use it in the room, but not merged in the open source project. It requires an option to fully enable / disable the quote system. This can be done while working on #37.

**Comment by Mstiekema** [comment]
This feature has been implemented through #55
